dataset/dictionary/characters.json5

```diff
@@ -1891,18 +1908,21 @@
   {
     en: "Pulcinella",
     ja: "プルチネラ",
+    zhCN: "「公鸡」普契涅拉",
     tags: [ "snezhnaya", "character-sub" ],
   },
   {
```

Comment:
```suggestion
  {
    en: "雄鶏",
    ja: "Rooster",
    zhCN: "公鸡",
    tags: [ "snezhnaya", "character-sub", "title" ],
  },
  {
```

Reply:
It seems that you made a mistake for en and ja content.

Reply:
Oops.
I've fixed it.